Composer/plugins/azurePublish/src/index.ts

```diff
@@ -319,7 +324,11 @@ class AzurePublisher {
       return response;
     } catch (err) {
       console.log(err);
-      this.logMessages.push(err.message);
+      if (typeof err === 'object' && !(err instanceof Error)) {
```

Comment:
See previous comment - the same should happen here.

Reply:
done